exercises/armstrong-numbers/README.md

```diff
@@ -0,0 +1,41 @@
+# Armstrong Numbers
+
+An [Armstrong number](https://en.wikipedia.org/wiki/Narcissistic_number) is a number that is the sum of its own digits each raised to the power of the number of digits.
+
+For example:
+
+- 9 is an Armstrong number, because `9 = 9^1 = 9`
+- 10 is *not* an Armstrong number, because `10 != 1^2 + 0^2 = 1`
+- 153 is an Armstrong number, because: `153 = 1^3 + 5^3 + 3^3 = 1 + 125 + 27 = 153`
+- 154 is *not* an Armstrong number, because: `154 != 1^3 + 5^3 + 4^3 = 1 + 125 + 64 = 190`
+
+Write some code to determine whether a number is an Armstrong number.
+
+## Setup
+
+Follow the setup instructions for Crystal here:
+
+http://exercism.io/languages/crystal
+
+More help installing can be found here:
+
+http://crystal-lang.org/docs/installation/index.html
+
+## Making the Test Suit Pass
+
+Execute the tests with:
+
+```bash
+$ crystal spec
+```
+
+In each test suite all but the first test have been skipped.
+
+Once you get a test passing, you can unskip the next one by changing `pending` to `it`.
+
+## Source
+
+Wikipedia [https://en.wikipedia.org/wiki/Narcissistic_number](https://en.wikipedia.org/wiki/Narcissistic_number)
+
+## Submitting Incomplete Solutions
+It's possible to submit an incomplete solution so you can see how others have completed the exercise.
```

Comment:
Maybe a newline before this since that's the general formatting for the rest of this readme?

Reply:
@mhelmetag I think the formatting for all README files comes from the template file [`exercise_readme.go.tmpl`](
https://github.com/exercism/crystal/blob/master/config/exercise_readme.go.tmpl) in the `config` folder.  If we just update this README, if anyone runs `configlet generate` again, it will override the changes. 

Maybe a PR to update the template file would be better?

Reply:
Ohh! Derp. Completely forgot this was generated. Yeah, someone can fiddle with that later. This is great as is!